docs/user-docs/openqa.md

```diff
@@ -0,0 +1,244 @@
+# TrenchBoot OpenQA user guide
+
+## OpenQA
+
+OpenQA is a tool for performing automated testing of operating systems developed
+by SUSE. It is used by multiple Linux distributions like OpenSUSE, Fedora and
+QubesOS.
+
+The results of TrenchBoot automated tests can be viewed on
+[the project's OpenQA instance][openqa-instance].
+
+## Usage
+
+The website allows anyone to freely monitor all the details about the tests run,
+including:
+- configuration variables
```

Comment:
For mkdocs, each list must be preceded with an empty line, otherwise it isn't treated as a list and is printed in one line.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/55/commits/7263a4b45e5309a2143428078989f11fac585021